File: submissions/sm_010_dipanshu/week_03/day_4/session_1/mirrorImage.js
function mirror(input){
    letters = 'abcdefghijklmnopqrstuvwxyz'
    var mirrorString = ''

    for(var i=0; i<input.length; i++){

        //checkfor uppercase
        if(input[i] == input[i].toUpperCase()){
            lower = input[i].toLowerCase()
            indexInLetters = letters.indexOf(lower)
            if(indexInLetters != -1){
                mirrorString += letters[25 - indexInLetters].toUpperCase()
            }
            else{
                mirrorString += input[i]
            }
        }

        //check for lowercase
        else{
            indexInLetters = letters.indexOf(input[i])
            if(indexInLetters != -1){
                mirrorString += letters[25 - indexInLetters]
            }
            else{
                mirrorString += input[i]
            }
        }
    }
    
    console.log(mirrorString)
}

mirror('AaBb-CcDd')

File: submissions/sm_115_suyash/week_13/day_4/cooding_1/mirror_image.py
print("Enter your string")
input_str = input()
upper_case = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
lower_case = "abcdefghijklmnopqrstuvwxyz"
mirror_str = ""

for char in input_str:
    flag = True
    for i in range(len(lower_case)):
        if lower_case[i] == char:
            flag = False
            mirror_str += lower_case[len(upper_case) - i - 1]
    if flag:
        for j in range(len(upper_case)):
            if upper_case[j] == char:
                flag = False
                mirror_str += upper_case[len(upper_case) - j - 1]

print(mirror_str)
